korean/okhome/okhome_success_story.txt

```diff
@@ -0,0 +1,2 @@
+Will be coming soon
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.